pipelines/ApVerifyWithFakes.yaml

```diff
@@ -0,0 +1,49 @@
+# Gen 3 pipeline for ap_verify
+# This concatenates various lsst.verify metrics to an AP pipeline
+
+description: Fully instrumented AP pipeline
+imports:
+    - location: $AP_PIPE_DIR/pipelines/ApPipe.yaml
+    - location: $AP_VERIFY_DIR/pipelines/MetricsRuntime.yaml
+    - location: $AP_VERIFY_DIR/pipelines/MetricsMiscWithFakes.yaml
+tasks:
+    createFakes:
+        class: lsst.ap.pipe.createApFakes.CreateRandomApFakesTask
+        config:
+            magMin: 20
+            magMax: 26
+            fraction: 0
+            fakeDensity: 2000
+    coaddFakes:
+        class: lsst.pipe.tasks.insertFakes.InsertFakesTask
+        config:
+            doSubSelectSources: True
+            sourceSelectionColName: "isTemplateSource"
+    visitFakes:
+        class: lsst.pipe.tasks.processCcdWithFakes.ProcessCcdWithFakesTask
+        config:
+            insertFakes.doSubSelectSources: True
+            insertFakes.sourceSelectionColName: "isVisitSource"
```

Comment:
Just to be sure: do you mean to have both this and regular CCD processing in the same pipeline?

Reply:
Yes, `ProcessCcdWithFakesTask` needs the fully calibrated calexps to exist.